cmd/login.go

```diff
@@ -44,27 +44,37 @@ func newLoginCmd() *cobra.Command {
 			"    $ pulumi login\n" +
 			"\n" +
 			"and this command will prompt you for an access token, including a way to launch your web browser to\n" +
-			"easily obtain one. You can script by using PULUMI_ACCESS_TOKEN environment variable.\n" +
+			"easily obtain one. You can script by using `PULUMI_ACCESS_TOKEN` environment variable.\n" +
 			"\n" +
-			"By default, this will log into app.pulumi.com. If you prefer to log into a separate instance\n" +
-			"of the Pulumi service, such as Pulumi Enterprise, specify a <url>. For example, run\n" +
+			"By default, this will log into `app.pulumi.com`. If you prefer to log into a separate instance\n" +
+			"of the Pulumi service, such as Pulumi Enterprise, specify a URL. For example, run\n" +
```

Comment:
I think `<url>` was referring to the optional argument to the command, but changing to more general `URL` sounds good to me as we show the command with the URL immediately below.